Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript

```diff
@@ -90,7 +107,8 @@ var defaultDirectives = function(context, path) {
     var sketchmigratePath = [[NSBundle mainBundle] pathForResource:@"sketchmigrate" ofType:nil inDirectory:@"sketchtool/bin"];
 
     var directives = [NSMutableDictionary dictionary];
-    [directives setObject:[[doc fileURL] path] forKey:@"path"];
+    [directives setObject:temporaryPath forKey:@"temporaryPath"];
+    [directives setObject:[[doc fileURL] path] forKey:@"path"]; // legacy
```

Comment:
We'll always need to send this info to allow users to see which file they exported from. So, I think we shouldn't mark it as 'legacy'.